app/src/main/java/es/wolfi/passman/API/Core.java

```diff
@@ -137,8 +137,8 @@ public static void requestAPI(Context c, String endpoint, RequestParams postData
 
         AsyncHttpClient client = new AsyncHttpClient();
         client.setBasicAuth(username, password);
-        client.setConnectTimeout(15000);
-        client.setResponseTimeout(15000);
+        client.setConnectTimeout(1000*15);      // 15s connect timeout
```

Comment:
Precalculated constants are better on this instances, perhaps this one is not one that is iterated very often, but calculating a constant is a waste of cpu in most cases, the comments are cool though

Reply:
fixed in 80ddfc55e03cfc5038788daff721dcf99c0e5bee